websites/O/Odies/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+      "$schema": "https://schemas.premid.app/metadata/1.0",
+      "author": {
+      "name": "bris",
+      "id": "656165298818514946"
+    },
+
+    "service": "Odies",
+    "description": {
+      "tr": "Odies, botlarınız ve siteleriniz için ücretsiz kod, yardım alabileceğiniz bir sitedir.",
+      "en": "Odies is a free code for your bots and sites, a site where you can get help."
+    },
+
+    "url": "odies.net",
+    "version": "1.0.7",
```

Comment:
v 1.0.0 on release

Reply:
![](https://bris.is-inside.me/MnJERFkt.png) ![](https://bris.is-inside.me/MnJERFkt.png) ![](https://bris.is-inside.me/KzI2WLOY.png) 

![](https://bris.is-inside.me/sbmJiilU.png) ![](https://bris.is-inside.me/T99aHkno.png) ![](https://bris.is-inside.me/olam2LmE.png)



**Too many blank lines at the end of the file fixed & Error page added &  Version arrangement has been made**